### PR TITLE
Update Ledger Polkadot JS Video link

### LINKED ISSUE #3048: Update links to Polkadot JS with Ledger video
With this video - https://youtu.be/7VlTncHCGPc

### PR DESCRIPTION
**Pull request by DrW3RK**
Found two instances with the old video. Also, no reference to the video on https://wiki.polkadot.network/docs/learn-ledger

### CODE REVIEW

File: docs/general/ledger.md [hunk 1]
```diff
@@ -82,6 +82,11 @@ At the end of the process you should have the newly installed Polkadot applicati
 
 ## Using on Polkadot-JS Apps
 
+If you are a beginner, please consider watching this video tutorial on how to connect your Ledger
```
Comment (salmad3) [inline docs/general/ledger.md]:
```suggestion
Please consider watching this video tutorial on how to connect your Ledger
```

File: docs/general/ledger.md [hunk 2]
```diff
@@ -82,6 +82,11 @@ At the end of the process you should have the newly installed Polkadot applicati
...
+device with Polkadot JS UI, with your accounts imported to Polkadot browser extension.
```
Comment (salmad3) [inline docs/general/ledger.md]:
```suggestion
device with the Polkadot-JS UI. Your accounts will need to be imported on the Polkadot-JS browser extension.
```

**Review by salmad3 — CHANGES_REQUESTED** [review]

File: docs/general/ledger.md (hunk 2 above)
Comment (profradha) [reply docs/general/ledger.md]: A better way to word it - (as the user can either import it to extension which is the recommended way or directly to the UI as explained in the later section)

```suggestion
device with Polkadot-JS UI, by importing your accounts to Polkadot-JS Extension
```

**Review by salmad3 — APPROVED** [review]
lgtm

**Pull request merged**